students/kulon_aleksander/lesson_05/chess_board.py

```diff
@@ -0,0 +1,10 @@
+n, m = input().split()
+tab = []
+for i in range(int(n)):
+    tab.append(['* '] * int(m))
+    for j in range(int(m)):
+        if i % 2 == 0 and j % 2 == 0:
+            tab[i][j] = '. '
```

Comment:
Instead of assigning ```tab[i][j] = '. ' ``` you can omit space at the end so it will be: ```tab[i][j] = '.' ``` and you can join values using space (line 10): ```print(' '.join(tab[i])) ```